astroquery/esasky/tests/test_esasky_remote.py

```diff
@@ -115,10 +118,15 @@ def test_esasky_query_object_maps(self):
                                          'ISO-IR', 'Herschel', 'JWST-MID-IR',
                                          'JWST-NEAR-IR', 'Spitzer'])
     def test_esasky_get_images(self, tmp_path, mission):
-        ESASky.get_images(position="M51", missions=mission, download_dir=tmp_path)
-
+        result = ESASky.get_images(
+            position="M51", missions=mission, download_dir=tmp_path
+        )
```

Comment:
keep these long, we use line length 120.

Reply:
I've merged the line triplets.